javatests/arcs/schematests/joins/JoinPlanTest.kt

```diff
@@ -0,0 +1,25 @@
+package arcs.schematests.joins
+
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import arcs.core.storage.keys.JoinStorageKey
+import arcs.core.storage.referencemode.ReferenceModeStorageKey
+import com.google.common.truth.Truth
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class JoinPlanTest {
+    @Test
+    fun canParseJoinStorageKey() {
+        DriverAndKeyConfigurator.configure(null);
+        val particle = ReadJoinPlan.particles.single()
+        val connection = particle.handles.values.single()
+
+        Truth.assertThat(connection.storageKey is JoinStorageKey)
```

Comment:
Import `com.google.common.truth.Truth.assertThat` instead, and then this is just `assertThat`

Reply:
Also:

```
assertThat(connetion.storageKey).isInstanceOf(...)
```

Reply:
Done! Thanks!